src/Account/Controllers/AccountViewController.php

```diff
@@ -135,6 +143,42 @@ public function success(): Response
         return $this->render('accounts/register_success.html.twig');
     }
 
+    /**
+     * @param Request $request
+     * @return Response
+     */
+    public function login(Request $request): Response
+    {
+        $form = $this->createLoginForm();
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $data = $form->getData();
+
+            try {
+                $user = $this->userRepository->getByEmail(new Email($data['email']));
+
+                if ($user && $user->getPassword() && $user->getPassword()->verifies($data['password'])) {
+                    if ($user->isActive()) {
+                        return $this->redirectToRoute('questions_view_index');
+                    }
+                    $this->addFlash('warning', $this->translator->trans('Account inactive'));
+                } else {
+                    $this->addFlash('danger', $this->translator->trans('Invalid credentials'));
+                }
+            } catch (\Exception $e) {
+                $this->addFlash('danger', $this->translator->trans('Login failed'));
```

Comment:
@stejes er is geen exception die we throwen, dus zou precies dit negeren en gewoon met de Symfony fout naar boven laten komen.